resources/sample_vaults/Tasks-Demo/Styling/Styling of Queries.md

```diff
@@ -0,0 +1,40 @@
+# Styling of Queries
+
+To test styling of queries, follow this:
+
+- [ ] **1. Open the Obsidian settings of the Demo vault and under Appearance | CSS Snippets, turn on `tasks-plugin-query-styling`.**
+
+- [ ] **2.** **Test 'group by' classes** - the following query result should have **red headings** named 'Priority 1: High' and 'Priority 4: Low'.
+
+```tasks
+path includes Styling of Queries
+group by priority
+```
+
+- [ ] **3. Test 'group by' classes #2** - the following should have a **black** heading named 'No due date':
+
+```tasks
+path includes Styling of Queries
+group by due
+```
```

Comment:
This one does work for me, in that the heading is black. But all the headings are black by default, so I am unsure whether this sounds as a success or not.

![image](https://user-images.githubusercontent.com/4840096/217620295-f52be30d-50ae-4334-8c91-8702dceca9fe.png)

Reply:
The rest worked.

Reply:
I'm using Obsidian 1.1.12.

I just tried using the build artefact `Tasks-Demo-VerifyCommit-Build2169-Run1` from this action, and got the same behaviour as my local build:
https://github.com/obsidian-tasks-group/obsidian-tasks/actions/runs/4127545531

I'm on Mac.

Reply:
OK - progress.

I changed the CSS to this - headings still black:

```css
.tasks-group-heading {
    color: red;
}
```

Then to this, and they turned read:


```css
.tasks-group-heading {
    color: red !important;
}
```

When I reverted to this, so the only change was the `!important`, they all went black again:

```css
.block-language-tasks:has(ul.plugin-tasks-query-result.tasks-group-by-priority) .tasks-group-heading {
    color: red !important;
}
```

Reply:
Sorry, I broke the threading with these comments... They were to address https://github.com/obsidian-tasks-group/obsidian-tasks/pull/1519#discussion_r1100645147